README.md

```diff
@@ -108,16 +157,30 @@ Now the cryoSPARC master process can be started by running
 
 ```cryosparcm start```
```

Comment:
On tier-1, I need to use ``` .\cryosparcm start ```

Reply:
`.\cryosparcm start` will only work if you are inside the installation directory of the cryosparc master. If you first run `source set_environment.sh`, this installation directory is prepended to `$PATH` and you can just use `cryosparc start`. This has been clarified in 20c2c2b0661171bad84f3c441b473db0b56fff68